pv_site_api/main.py

```diff
@@ -417,6 +460,104 @@ def get_pv_estimate_clearsky_many_sites(
     return res
 
 
+@app.get("/enode/link", response_class=RedirectResponse)
+def get_enode_link(
+    redirect_uri: str, session: Session = Depends(get_session), auth: Auth = Depends(auth)
+):
+    """
+    ### Returns a URL from Enode that starts a user's Enode link flow.
+    """
+    if is_fake():
+        return make_fake_enode_link_url()
+
+    client = session.query(ClientSQL).first()
```

Comment:
I think we should add the "TODO" pointing to the issue to fix this to all of those.